packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/QuickActions.tsx

```diff
@@ -30,15 +30,16 @@ const DesktopContainer = styled(WebContainer)`
     display: grid;
     grid-template-columns: 1fr 1fr;
     gap: ${spacingsPx.md};
+    padding: ${spacingsPx.xs};
     border-top: 1px solid ${({ theme }) => theme.borderOnElevation0};
 
     ::after {
         content: '';
         position: absolute;
         top: ${spacingsPx.xxs};
+        bottom: ${spacingsPx.xxs};
         right: 50%;
         width: 1px;
-        height: 100%;
```

Comment:
This created additional tiny scrolling for the whole sidebar which was not necessary